hosts.ini

```diff
@@ -0,0 +1,9 @@
+[all]
+gnbsimNode1 ansible_host=172.16.193.6 ansible_user=aether ansible_password=aether ansible_sudo_pass=aether
+onramp1 ansible_host=172.16.41.103 ansible_user=aether ansible_password=aether ansible_sudo_pass=aether
+
+[srsran_nodes]
+gnbsimNode1
```

Comment:
Should not this be called `srsRanNode1`?

Reply:
done